src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -24,8 +24,8 @@
 class SyliusApiExtension extends AbstractResourceExtension implements PrependExtensionInterface
 {
     protected $configFiles = array(
```

Comment:
 @stloyd I think it is better to start with something simple, I need that for other PR.